styleguide/source/_meta/_00-head.twig

```diff
@@ -28,5 +28,4 @@
 
   </head>
   <body class="{{ bodyClass }}">
-    <div class="ama-theme">
-        <div class="nav-primary_overlay-mobile"></div>
+    <div class="nav-primary_overlay-mobile"></div>
```

Comment:
This class should be updated to traditional BEM standards from the modified BEM.

Reply:
Get rid of the `<div class="nav-primary_overlay-mobile"></div>` - we don't need that yet